core/import_map.ts

```diff
@@ -54,6 +54,17 @@ export class ImportMap implements ImportMapInterface {
       }
     }
 
+    const imports = Object.entries(importMap.imports)
+      .filter(
+        ([key, val]) =>
+          (key.endsWith("/") && val.endsWith("/")) ||
+          (!key.endsWith("/") && !val.endsWith("/"))
```

Comment:
nice work!

But this is just filtering out invalid rules.

Could you add the test to make sure the rules like this work fine?

```json
{
  "imports": {
    "moment": "/node_modules/moment/src/moment.js",
    "moment/": "/node_modules/moment/src/",
    "lodash": "/node_modules/lodash-es/lodash.js",
    "lodash/": "/node_modules/lodash-es/"
  }
}
```

This requires adding test cases to `module_resolver.test.ts`

Reply:
Done!